datagenerator/src/main/resources/rythm/enum.rythm

```diff
@@ -0,0 +1,21 @@
+@import com.linkedin.pegasus.generator.spec.ClassTemplateSpec
+@args String className
```

Comment:
As I commented before, we should try not passing too many parameters to rythm template, instead can pass a single variable (for example, EnumTemplateSpec or EnumDataSchema) here and access their fields in rythm for easy reading.